src/containers/CallLogs/CallLogs.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getAllCalls } from '../../store/actions/callActions';
+import CallLog from '../../components/CallLog/CallLog.jsx';
+
+const mapStateToProps = state => {
+  return {
+    calls: state.callLogReducer.calls
+  };
+};
+
+class CallLogs extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    this.props.getAllCalls();
+  }
+
+  singleCallHandler(e) {
+    const callId = e.target.dataset.id;
```

Comment:
already this assumes that whatever you click will have data-id. 
I'd suggest just passing in the id directly to the function invocation